test/reporters/dot.spec.js

```diff
@@ -80,28 +86,38 @@ describe('Dot reporter', function() {
           runner = createMockRunner('pass', 'pass', null, null, test);
           Dot.call({epilogue: function() {}}, runner);
           process.stdout.write = stdoutWrite;
-          var expectedArray = ['\n  ', Base.symbols.dot];
+          var expectedArray = ['\n  ', 'fast_' + Base.symbols.dot];
           expect(stdout, 'to equal', expectedArray);
         });
       });
     });
     describe('if window width is equal to or less than 1', function() {
       describe('if test speed is fast', function() {
-        it('should return a dot', function() {
+        it('should return a grey dot', function() {
           runner = createMockRunner('pass', 'pass', null, null, test);
           Dot.call({epilogue: function() {}}, runner);
           process.stdout.write = stdoutWrite;
```

Comment:
```js
          expect(test.speed, 'to equal', 'fast');
```

Reply:
I don't understand what you mean and why we check only speed.

Reply:
the field is set for `pass` events by `runner` in `Base` ctor; it's the only field set there.
`test.speed` assertions are _in addition to_ your `stdout` assertions, not in lieu of them.